content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -30,29 +54,35 @@ On most Kubernetes clusters, cAdvisor exposes a simple UI for on-machine contain
 
 The Kubelet acts as a bridge between the Kubernetes master and the nodes. It manages the pods and containers running on a machine. Kubelet translates each pod into its constituent containers and fetches individual container usage statistics from cAdvisor. It then exposes the aggregated pod resource usage statistics via a REST API.
 
-## Metrics-Server
-
-metric-server acts as a short-term (single-data-point) in-memory store for CPU and memory metrics.  It's designed to be a lightweight solution for providing a small set of metrics to system components.
-
 ## Full Metrics Pipelines
 
-Many full metrics solutions exist for Kubernetes. The following two are popular solutions:
+Many full metrics solutions exist for Kubernetes. Prometheus and Google Cloud
+Monitoring are two of the most popular.
 
 ### Prometheus
 
-[Prometheus](https://prometheus.io) is able to natively monitor Prometheus, and the [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/) can help simplify setup
-on Kubernetes.  It also has support for serving the custom metrics API via the [Prometheus adapter](https://github.com/directxman12/k8s-prometheus-adapter).  Prometheus itself provides
-a robust query language for examining your data, a built-in dashboard, and is also a [supported data source for Grafana](https://prometheus.io/docs/visualization/grafana/).
+[Prometheus](https://prometheus.io) natively monitors Prometheus.
+The [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/)
+simplifies Prometheus setup on Kubernetes, and allows you to serve the
+custom metrics API using the
+[Prometheus adapter](https://github.com/directxman12/k8s-prometheus-adapter).
+Prometheus provides a robust query language and a built-in dashboard for
+querying and visualizing your data. Prometheus is also a supported
+data source for [Grafana](https://prometheus.io/docs/visualization/grafana/).
 
 ### Google Cloud Monitoring
 
-Google Cloud Monitoring is a hosted monitoring service that allows you to visualize and alert on important metrics in your application. It uses Heapster to automatically push all collected metrics to Google Cloud Monitoring. These metrics are then available in the [Cloud Monitoring Console](https://app.google.stackdriver.com/). This storage backend is the easiest to setup and maintain. The monitoring console allows you to easily create and customize dashboards using the exported data.
+Google Cloud Monitoring is a hosted monitoring service you can use to
+visualize and alert on important metrics in your application. It uses
+Heapster to collect metrics from Kubernetes, and gives access to them
```

Comment:
this should say "it uses a forked copy of Heapster" or just drop the mention of Heapster entirely